### PR TITLE
Marble1280

### PR DESCRIPTION
**Pull request by danthewolfe**

### CODE REVIEW

File: site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import typy from 'typy'
+
+export const Form = ({ closeFunc }) => {
+  // const [content, setContent] = useState(<Loading />)
+  const [name, changeName] = useState('')
+  const [email, changeEmail] = useState('')
+  const [feedback, changeFeedback] = useState('')
+  const [patching, setPatching] = useState(false)
+  const emailRegex = /^[\w-.]+@([\w-]+.)+[\w-]{2,4}$/g
+
```
Comment (danthewolfe) [inline site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js:23]: Add
```
const [response, setResponse] = useState(null)

File: site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  const {
+    handleSubmit,
+  } = useForm(submit)
+
+  function submit () {
+    console.log('Submitted')
+  }
+
+  if (!typy(createData.successFunc, 'number').safeString === '') {
```
Comment (danthewolfe) [inline site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js:32]:
```
if (!response) {
```

File: site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                setPatching(true)
+                const body = {
+                  name: name,
+                  email: email,
+                  feedback: feedback,
+                }
+                createData({
+                  body: body,
+                  successFunc: (data) => {
+                    // closeFunc()
```
Comment (danthewolfe) [inline site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js:91]: Add the following:
```
setResponse(true)
setPatching(false)
```

File: site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js (hunk 2 above)
Comment (pweikel) [reply site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js]: Changed!

File: site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js (hunk 3 above)
Comment (pweikel) [reply site/src/@ndlib/gatsby-theme-marble/components/Internal/FeedbackForm/index.js]: Changed!

**Pull request closed without merging**